atomic_reactor/plugins/generate_sbom.py

```diff
@@ -285,6 +289,37 @@ def get_unique_and_sorted_components(
 
         return unique_components
 
+    def push_sboms_to_registry(self):
+        docker_config = os.path.join(self.workflow.conf.registries_cfg_path, '.dockerconfigjson')
+
+        if not os.path.exists(docker_config):
+            self.log.warning("Dockerconfig json doesn't exist in : '%s'",  docker_config)
+            return
+
+        tmpdir = tempfile.mkdtemp()
```

Comment:
Do we need to keep the sbom files for a while after push?

Reply:
it is running in container in openshift, we don't really care about such small files